project/ReactiveMongo.scala

```diff
@@ -130,20 +130,19 @@ object Dependencies {
     "org.specs2" %% "specs2-core" % specsVer.value % Test
   }
 
-  val slf4jVer = "1.7.12"
+  val slf4jVer = "1.7.21"
   val log4jVer = "2.5"
 
   val slf4j = "org.slf4j" % "slf4j-api" % slf4jVer
-  val slf4jSimple = "org.slf4j" % "slf4j-simple" % slf4jVer
 
   val logApi = Seq(
-    slf4j % "provided",
```

Comment:
Not desired

Reply:
@cchantep Happy to return to a provided dependency, however you should consider removing the Log4J one as soon as, because it's causing logs to fail for most clients, only partial logs show.

Reply:
- Most of these dependencies are `Test`-only, so there is no impact on the runtime of the project using the lib.
- The `provided` semantic is appropriate for `slf4j` (and is on-purpose)